packages/core/src/modules/sd-jwt-vc/__tests__/sdjwtvc.fixtures.ts

```diff
@@ -0,0 +1,17 @@
+export const simpleJwtVc =
```

Comment:
Are these just created by the framework? If not, would be good to link to the source for where they were found.

Reply:
Yes created by the framework. Would be good to have external fixtures as well